src/importexport/mei/internal/meireader.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "meireader.h"
+
+#include "meiimporter.h"
+
+#include "libmscore/masterscore.h"
+#include "engraving/engravingerrors.h"
+#include "translation.h"
+
+using namespace mu::iex::mei;
+using namespace mu::engraving;
+
+mu::Ret MeiReader::read(MasterScore* score, const io::path_t& path, const Options& options)
+{
+    if (!fileSystem()->exists(path)) {
+        return make_ret(Err::FileNotFound, path);
+    }
+
+    MeiImporter importer(score);
+    if (!importer.read(path.toQString())) {
```

Comment:
...which will allow you to eliminate this `toQString` conversion

Reply:
Oops, it looks like this one is still there

Reply:
Hopefully fixed...